### ISSUE TITLE
vanilla doors  don't work on bukkit server!

### ISSUE THREAD

**Issue by Ironic8b498** [Ordinastie/MalisisDoors#261]
I have a combo Spigot/bukkit and sponge/forge server and would like to add MalisisDoors to the forge server but if I do the players would have to change clients to play on the other servers, is this a bug or a issue that was not thought of yet?

**Comment by ZombieHDGaming** [comment]
If they have mods on the client, they can still connect to the Spigot/Bukkit server, but the normal vanilla client cannot connect to the Forge server. If they are having issues connecting to the Spigot/Bukkit server with MalisisDoors added, have them disable the config option that overwrites the vanilla doors and trapdoors.

**Comment by Ironic8b498** [comment]
that setting allows me to play on all the servers, with no issues so far,Thanks

**Comment by Ironic8b498** [comment]
One question, since I'm new to Malisis doors does overwriting the vanilla doors and trapdoors change anything on the forge server?

**Comment by ZombieHDGaming** [comment]
Yes, it overwrites door behaviors making it non-vanilla in a sense. It doesn't need to be enabled, but it gives a better effect with MalisisDoors
